.gitignore

```diff
@@ -1,4 +1,5 @@
 *.py[ocd]
 *.sqlite3
 !example.env
-.env
\ No newline at end of file
+.env
+*.node_modules/
```

Comment:
Missing end of file newline in this and other files.
Not too big of a deal right now, but do install the editorconfig addon in your editor of choice for later.